src/service/chat.js

```diff
@@ -143,9 +143,11 @@ const sendSignedDMOtherProfile = async ({user_id, message}) => {
   return Promise.reject(response.data?.data);
 };
 
-const getOrCreateAnonymousChannel = async (userId) => {
+const getOrCreateAnonymousChannel = async (userId, oldChannelId = null, context = null) => {
   const payload = {
-    members: [userId]
+    members: [userId],
+    oldChannelId,
+    context
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `oldChannelId` and `context` as parameters to the `getOrCreateAnonymousChannel` function is a good change, but it's important to ensure that all calls to this function are updated to pass these parameters where necessary. If not, it could lead to unexpected behavior.

```diff
- const getOrCreateAnonymousChannel = async (userId) => {
+ const getOrCreateAnonymousChannel = async (userId, oldChannelId = null, context = null) => {
    const payload = {
     members: [userId],
+    oldChannelId,
+    context
   };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->